src/pages/ContactPage.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <h1>This is Contact Page</h1>
+</template>
+
+<style lang="scss">
+h1 {
+  font-family: $ff-primary;
+  font-size: $fs-super-large;
+  text-align: center;
+  margin-top: 55px;
+}
```

Comment:
You are styling the `h1` in multiple components here and re-using those styles.
It would be a good choice to refactor it in some way - we might even introduce a component for the headers of all the pages, maybe something like `PageHeader` 🤔 and have the styles there.
Then we could just import it where we need it and vouila!
What do you think? 😄